platform/kube/inject/testdata/auth.cert-dir.yaml.injected

```diff
@@ -49,10 +49,16 @@ spec:
         securityContext:
           runAsUser: 1337
         volumeMounts:
+        - mountPath: /etc/istio
```

Comment:
if the certs and config are written and updated differently it's better to have different paths